README.md

```diff
@@ -220,6 +221,45 @@ C:\Users\bion\Desktop>dumpbin /IMPORTS .\example.exe | findstr msvcp
     msvcp140d_oss.dll
 ```
 
+# How To Run The Tests
+
+In order to run any of the tests you must be in a correctly configured environment for the architecture you are
+targeting, have python 3.8 or newer and have `clang-cl` on the PATH. If you are testing the x64 flavor of the STL then
+running from the "x64 Native Tools Command Prompt" is sufficient for setting up the environment.
+
+## Running All The Tests
+
+After configuring the project, running `ctest` from the root of the build output directory will run all the tests.
+CTest will only display the standard error output of tests which failed. In order to get more details from CTest's
+`lit` invocations, run the tests with `ctest -V`.
+
+## Running A Subset Of The Tests
+
+In order to run a subset of one of the testsuites you must invoke `lit` yourself and point it to the desired
+subdirectory of tests.
+
+The CMake configuration step produces a Python script, `${PROJECT_BINARY_DIR}\tests\llvm-lit\llvm-lit.py`.
+`llvm-lit.py` simply calls into the standard `lit` found in `llvm-project\llvm\utils\lit` with certain options
+pre-configured. Namely a mapping of `lit.cfg` files to `lit.site.cfg` files is provided, which informs lit which
+site-specific config file to load whenever it discovers a general config file.
+
+While one can directly use `lit` to run our tests by hand, we only support using the generated `llvm-lit`.

```

Comment:
I am confused by this entire section. The first paragraph said "you must invoke `lit` yourself", but the third paragraph says that directly using `lit` is unsupported, and all of the examples demonstrate `llvm-lit.py`.

Additionally, users of the test infrastructure shouldn't care about how information is communicated to the underlying `lit`. (This seems like it would be more appropriate as comments in the test infrastructure, or a README in a subdirectory of the test infrastructure.)

Should this section simply explain that `llvm-lit.py` can be invoked with a test suite directory (like `tests\std`), a test category directory (e.g. libcxx's nicely organized subdirectories), or an individual test directory?

Reply:
I think I'll add a little bit more context in the examples and see what you think.